src/looper.hxx

```diff
@@ -70,7 +70,7 @@ class Looper : public AudioProcessor, public TimeObserver
     int fpb;
     
     //vector<float> tmpRecordBuffer;
-    LooperClip* clips[10];
+    LooperClip** clips = new LooperClip*[nscenes];
```

Comment:
Hmm, be careful here - this is allocating memory (aka implementation) in the header file.. that's not a good idea. Does this generate a warning on compilation?